Sources/ApiVideoPlayer/SwiftUI/ApiVideoPlayer.swift

```diff
@@ -128,6 +128,7 @@ public struct ApiVideoPlayer: UIViewControllerRepresentable {
 }
 
 struct SwiftUIView_Previews: PreviewProvider {
+    @available(iOS 13.0.0, *)
```

Comment:
As SwiftUI only exists on iOS > 13.0, could you add this flag on all SwiftUI classes?

Reply:
Shouldn't `ApiVideoPlayer` (SwiftUI) be under this flag?